cmake/Modules/FunctionQbtCXXCompilerAndModeCheck.cmake

```diff
@@ -1,24 +0,0 @@
-# Function for ensuring the build does not use a lower than the required minimum C++ mode, _min_std.
```

Comment:
Isn't a good candidate for extracting?

Reply:
@glassez same reasoning as  https://github.com/qbittorrent/qBittorrent/pull/13327#discussion_r485157109.